libs/styles/styles.css

```diff
@@ -1379,7 +1399,7 @@ a.xl-link {
 
 a {
   color: var(--link-color);
-  text-decoration: none;
```

Comment:
I'm not very convinced with removing this, it adds underline for links in the bottom part of the footer and the region modal, maybe we can adapt this closer to where we actually need it, like the FAQ / Editorial Card sections.

Reply:
Yea the reason I removed this is because the Spectrum design system specified that the links should be underlined by default. It seems like the footer, bottom footer, and the region modal are special cases where it shouldn't be applied according to the mockups. It made sense to handle these specific cases individually rather than overriding the default styling globally. What do you think about this approach? Or do you think we should only add the underline to the FAQ/ Editorial cards for the purposes of the demo page?
<img width="836" alt="Screenshot 2024-08-07 at 11 29 56 PM" src="https://github.com/user-attachments/assets/1866746f-b7a5-4447-b9a9-91ad8d7c4a19">

Reply:
If the new design decision is to underline, then I think what you're suggesting is better for now, remove text decoration where it's not needed.